cmd/src/bin/tikv-ctl.rs

```diff
@@ -2263,7 +2263,13 @@ fn run_ldb_command(cmd: &ArgMatches<'_>, cfg: &TiKvConfig) {
         None => Vec::new(),
     };
     args.insert(0, "ldb".to_owned());
-    let opts = cfg.rocksdb.build_opt();
+    let key_manager = DataKeyManager::from_config(&cfg.encryption, &cfg.storage.data_dir)
```

Comment:
What if `cfg` is a default value?

Reply:
considered the case

Reply:
Just store files without encryption

Reply:
Can commands like `ldb repair` ruin the data when dealing encrypted db without keys?

/cc @Connor1996

Reply:
Not familiar with encrypted db. /cc @yiwu-arbug

Reply:
@BusyJay haven't test, but it shouldn't, as `DataKeyManager` will fail to initialize and panic in wrong key is provided.